internal/rbd/manager.go

```diff
@@ -66,8 +66,8 @@ func (mgr *rbdManager) Destroy(ctx context.Context) {
 	}
 }
 
-// getCredentials sets up credentials and connects to the journal.
-func (mgr *rbdManager) getCredentials() (*util.Credentials, error) {
+// GetCredentials sets up credentials and connects to the journal.
```

Comment:
GetCredentials should be a internal function. can you add a method to the manager that requires the creds itself instead of exposing the creds outside of manager?